app/src/main/java/org/mozilla/fenix/library/history/viewholders/HistoryListItemViewHolder.kt

```diff
@@ -43,14 +47,12 @@ class HistoryListItemViewHolder(
         }
     }
 
+    @Suppress("UndocumentedPublicFunction")
```

Comment:
I think we can just this without adding the `BindingParams`. There's also no need to change this since you can call the FeatureFlag from here.

Reply:
See https://github.com/mozilla-mobile/fenix/pull/24374#discussion_r847555086. If you decide to continue this route, I would ask you to drop the `BindingParams` data class in favor of suppressing long parameter list. I would also rename `syncedHistoryVisible` to `showSyncedHistory` to be consistent with the other parameter naming. Also, we expect public class/functions to be documented.